pkgs/development/python-modules/fastdotcom/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  icmplib,
+  pythonOlder,
+  requests,
+}:
+
+buildPythonPackage rec {
+  pname = "fastdotcom";
+  version = "0.0.6";
+  format = "setuptools";
```

```suggestion
  pyroject = true;
```

And set up `bulld-system` instead.